music_assistant/controllers/players.py

```diff
@@ -1113,6 +1113,8 @@ async def add_currently_playing_to_favorites(self, player_id: str) -> None:
                     raise PlayerCommandFailed("No current item to add to favorites")
                 # send the streamtitle into a global search query
                 search_artist, search_title_title = stream_title.split(" - ", 1)
+                # strip off the enhanced text added by the Radio Paradise provider
+                search_title_title = search_title_title.split("|")[0].strip()
```

Comment:
We should maybe update this cod enow that we have stream_metadata as replacement for the old stream_title.
I dont like having provider specific code in core controllers but this can be generalized a bit